ios/Sources/SentryCapacitorPlugin/SentryCapacitorPlugin.swift

```diff
@@ -70,8 +70,7 @@ public class SentryCapacitorPlugin: CAPPlugin, CAPBridgedPlugin {
         }
 
         do {
-            let options = try Options.init(dict: optionsDict)
-
+            let options = try SentryOptionsInternal.initWithDict(optionsDict)
```

Comment:
I now realized that Capacitor is already in Swift :)

Reply:
it has been like that for ages hehe